src/components/Todo.tsx

```diff
@@ -0,0 +1,57 @@
+import { RiDeleteBin2Fill, RiFileListLine } from 'react-icons/ri'
+import { Component } from 'react'
+import { Link } from 'react-router-dom'
+
+interface IPropsValues {
+  key: number
+  userId: number
+  id: number
+  title: string
+  body: string
+  index: number
+  delete: (index: number, value: number) => void
+  toggleTodo: (id: number, changeUserId: number) => void
+}
+
+class Todo extends Component<IPropsValues, { changeChecked: boolean }> {
```

Comment:
Единообразное название

```suggestion
class Todo extends Component<IProps, IState> {
```